aws-example-network/README.md

```diff
@@ -33,8 +33,6 @@ You must have these secrets on your Cloudify Manager `tenant`:
 
   * `aws_access_key_id`: Your AWS credentials access key ID.
   * `aws_secret_access_key`: Your AWS credentials access key secret.
-  * `aws_region_name`: The EC2 region, for example `us-east-1`.
-  * `ec2_region_endpoint`: The EC2 region service endpoint, for example `ec2.us-east-1.amazonaws.com`.
   * `aws_availability_zone`: Your preferred availability zone in your EC2 region, such as `us-east-1c`.
```

Comment:
Zone should also not be a secret. Make it input as well.

Reply:
its already done in this PR, I will fix the documentation.